speech-assets/IntentSchema.json

```diff
@@ -105,6 +109,46 @@
           "type": "AMAZON.NUMBER"
         },
         {
+          "name": "BinaryLogicOpOne",
+          "type": "BINARY_LOGIC_OPERATOR"
+        },
+        {
+          "name": "ComparisonColumnTwo",
+          "type": "COLUMN_NAME"
+        },
+        {
+          "name": "ComparatorTwo",
+          "type": "COMPARATORS"
+        },
+        {
+          "name": "ColumnValueTwo",
+          "type": "COLUMN_VALUE"
+        },
+        {
+          "name": "ColumnNumberTwo",
+          "type": "AMAZON.NUMBER"
+        },
+        {
+          "name": "BinaryLogicOpTwo",
+          "type": "BINARY_LOGIC_OPERATOR"
+        },
+        {
+          "name": "ComparisonColumnThree",
+          "type": "COLUMN_NAME"
+        },
+        {
+          "name": "ComparatorThree",
```

Comment:
Because refine only has one format/structure (there's no other clauses of
the query to make combinations with) adding these additional ones just add
two more samples. Check out the sample utterances it'll be clearer

On Friday, April 22, 2016, Gabe Lyons <notifications@github.com> wrote:

> In speech-assets/IntentSchema.json
> <https://github.com/vqtran/EchoQuery/pull/48#discussion_r60758583>:
>
> > +          "type": "COLUMN_VALUE"
> > +        },
> > +        {
> > +          "name": "ColumnNumberTwo",
> > +          "type": "AMAZON.NUMBER"
> > +        },
> > +        {
> > +          "name": "BinaryLogicOpTwo",
> > +          "type": "BINARY_LOGIC_OPERATOR"
> > +        },
> > +        {
> > +          "name": "ComparisonColumnThree",
> > +          "type": "COLUMN_NAME"
> > +        },
> > +        {
> > +          "name": "ComparatorThree",
>
> how does that not demolish our sample utterances
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/vqtran/EchoQuery/pull/48/files/eb24dfba8821febd34156507e5cbc5b2d9568404#r60758583>
>